doc/connectivity/bluetooth/api/mesh/dfu.rst

```diff
@@ -170,6 +170,11 @@ Firmware metadata
 |                        |              | with a status message.                 |
 +------------------------+--------------+----------------------------------------+
 
+  .. note::
+
+      In Mesh DFU metadata, AES-CMAC serves as a hashing function with a fixed key, not for
+      encryption. The resulting hash isn't secure due to the key being known.
```

Comment:
```suggestion
  .. note::

      The AES-CMAC algorithm serves as a hashing function with a fixed key and is not used for
      encryption in Bluetooth Mesh DFU metadata. The resulting hash is not secure since the key is known.
```